rethinking-css.md

```diff
@@ -1,24 +1,24 @@
 # Rethinking CSS
 
-In this article, I hope to define what I feel are the biggest issues with CSS, and ultimately, how those issues can be overcome.
+In this article I hope to define what I feel are the biggest issues with CSS and, ultimately, how those issues can be overcome.
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+<!-- DON’T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
Revert in case it ruins doctoc

Reply:
I reverted this at some point. I blame your force pushing ;)